tests/schema.php

```diff
@@ -0,0 +1,12 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Abstract schema for CakePHP tests.
+ *
+ * This format resembles the existing fixture schema
+ * and is converted to SQL via the Schema generation
+ * features of the Database package.
+ */
+return [
+];
```

Comment:
I'm planning to generate the contents of this file using the existing fixture definitions, but I'll do that in a separate pull request.